pkgs/build-support/pkg-config-wrapper/default.nix

```diff
@@ -87,13 +96,15 @@ stdenv.mkDerivation {
       printWords ${pkg-config} > $out/nix-support/propagated-user-env-packages
     ''
 
-    + optionalString propagateDoc ''
+    + optionalString propagateDoc (''
       ##
-      ## Man page and info support
+      ## Man page and doc support
       ##
```

Comment:
Put the comment outside the builder so it doesn't cause a rebuild in the future, perhaps?

Reply:
Indeed! And at some point somebody should do the same for `cc-wrapper` and `bintools-wrapper`---I put the `##` in those before I learned to regret it.